man/zeek.8

```diff
@@ -148,6 +150,28 @@ Output file for script execution statistics
 .TP
 .B ZEEK_DISABLE_ZEEKYGEN
 Disable Zeekygen (Broxygen) documentation support
+.SH OUTPUT FORMAT
+Output is written in multiple files depending on configuration. Default
+location is the current directory. Packets can be written to a tcpdump file.
+
+The output written by Zeek can be formatted in multiple ways using the
+logging framework.
+.PP
+The default are files in human-readable (ASCII) format and data is organized
+into columns (tab-delimited), They can be processed by the \fBzeek-cut\fR tool.
+
+
+.SH EXAMPLES
+Read a capture file:
```

Comment:
perhaps "Read a capture file and generate the default logs:"

Though that is more of a style question - I am fine with the current formulalation too.

Reply:
Agree and updated.

I ended up having made the changes in my kramse/zeek master branch :-( but hope it works out anyway.